authors/apps/authentication/serializers.py

```diff
@@ -16,6 +18,22 @@ class RegistrationSerializer(serializers.ModelSerializer):
         write_only=True
     )
 
+    def validate_username(self, data):
```

Comment:
Hi @unah254 good work but I think this can be reduced to one regex.

Reply:
Hello, thanks let me change